server/views/pages/additional-learning-needs-screener/add-challenges/index.njk

```diff
@@ -32,7 +32,7 @@
                     {% for challengeType in challengeTypes %}
                       <div class="govuk-checkboxes__item">
                         {% set fieldId = 'challengeTypeCodes' ~ (loopIndex if loopIndex > 0) %}
-                        <input class="govuk-checkboxes__input" id="{{ fieldId }}" name="challengeTypeCodes" type="checkbox" value="{{ challengeType.code }}">
+                        <input class="govuk-checkboxes__input" id="{{ fieldId }}" name="challengeTypeCodes" type="checkbox" value="{{ challengeType.code }}" {{ 'checked' if form.challengeTypeCodes.includes(challengeType.code) }}>
```

Comment:
Forgot this when I did this page first!
Without this change, when you come back to this page having click Change on Check Your Answers, none of the checkboxes are selected with your previous selections! 🙄